cli/tests/e2e/test_ci.py

```diff
@@ -195,7 +223,7 @@ def git_tmp_path_with_commit(monkeypatch, tmp_path, mocker):
         check=True,
         capture_output=True,
     )
-    subprocess.run(["git", "pull", "origin"])
+    subprocess.run(["git", "fetch", "origin"])
```

Comment:
`pull` throws warnings here because there is no branch checked out in this new repo we created, `fetch` is less distracting for people trying to debug because it removes the meaningless error.